cloudbuild.yaml

```diff
@@ -13,15 +13,62 @@
 # limitations under the License.
 
 steps:
+  - name: 'bash'
```

Comment:
The change is good, but I'm wondering if the cloudbuild/Make parallel infrastructure is starting to get out of hand. Just thinking out loud.

Maybe we could just have a single Dockerfile that exposes Make as an entrypoint? Then you would have:

Dockerfile:
FROM (whatever)
(install deps)
WORKDIR /work

Makefile.real:
... all our actual rules ...

Makefile:
%:
   @docker build -t gvisor-website . && docker run -v $PWD:/work gvisor-website make -f Makefile.real $@

cloudbuild.yaml:

steps:
  - name: 'gcr.io/builders/docker'
    args: ['build', '-t', 'gvisor-website', '.']
  - name: 'gcr.io/builders/docker'
    args: ['run', 'gvisor-website', 'make', '-f', 'Makefile.real', 'deploy']

Reply:
(Not something that should be pushed as part of this change.)

Reply:
Yeah, I thought about that too. Just haven't had a good idea about how to fix it. There is https://github.com/GoogleCloudPlatform/cloud-build-local command that you can use to run cloudbuild.yaml steps in a local Docker, which I considered using.

It kind of defeats the purpose of having different build steps if we have to have a custom container image that contains all the bits needed for every step (not that splitting them up gets us much anyway), and this isn't much different than just having a custom built image that we use from cloud build. The difference being caching, vs. not caching the intermediate image.

i.e.

Dockerfile:
```
FROM (whatever)
(install make, bazel, go, git, etc.)
ENTRYPOINT make
```

```
$ docker build -t gcr.io/gvisor-website-ci/make . & docker push gcr.io/gvisor-website-ci/make
```

cloudbuild.yaml
```
steps:
  - name: 'gcr.io/gvisor-website-ci/make
  - name: 'gcr.io/gvisor-website-ci/make
    args: ['deploy']
```